### DISCUSSION TITLE
using copy to clipboard or exporting a simulation to excel: Pksim v9.1

### DISCUSSION THREAD

**Discussion by amitvtaneja** [Open-Systems-Pharmacology/Forum#1215] [Questions & Problems]
Dear colleagues

I have a couple of questions regarding the use of the copying to clipboard or exporting the simulated data feature for a particular simulation- right click on the chart area under the tab "Time profile Analysis" 

1. When copying to clipboard, the figure has a grey border that is not very aesthetically appealing. Is there anyway i can alter the colour/appearance of this border?  
2. In case I export the simulation to pdf, in the pdf output all the observed data points overlaid on the simulation disappear and only the simulated curves are visible in the figure. 

Any advice on the aforesaid is welcome. Unfortunately i cannot share the graphs for now  given confidentiality reasons.

Thanks

Amit Taneja
Galapagos Pharma 
![image](https://user-images.githubusercontent.com/20029513/170236027-0d612c6b-c49c-4efd-a03c-79c2bf338d7f.png)

**Comment by StephanSchaller** [comment]
Dear Amit,

the grey border can be removed by changing the color here:
![image](https://user-images.githubusercontent.com/25477643/170245315-2f200f88-f05d-4923-9fc3-2be174a1af52.png)

Edit: For the PDF Report, I believe this issue was reported as a bug. Unfortunately, due to challenges with updating the PDF reports creation for new features and releases (LaTeX), these have been discontinued for the newest release (V11) and may be replaced by markdown-based solutions in the future.

Best,
Stephan
